net/cloudflared/files/cloudflared.config

```diff
@@ -4,7 +4,10 @@ config cloudflared 'config'
 	option token ''
 	option config '/etc/cloudflared/config.yml'
 	option origincert '/etc/cloudflared/cert.pem'
+	option edge-bind-address 'auto'
```

Comment:
so again I missed this, hyphen is not allowed for option name and now it causes an error. You should use underscore here. Is it really "run tested"?

Reply:
and is it really can be "auto"?

Reply:
Yes, but also no... At first I only added `protocol`, just as a refactor of another pr. Then I decided to add more parameters and didn't tested the change...  
Yes, it can be auto, I did read the docs [here](https://developers.cloudflare.com/cloudflare-one/connections/connect-networks/configure-tunnels/tunnel-run-parameters/#edge-ip-version)

Reply:
Sir what should I do next to make up for my mistake 😭  
Must be very very careful to make contribution, not destroy everything with "kindness"  
I made some changes, but I'm not sure if I should create a new commit or do `git commit --amend` as before. Seems a new pull request is required.  
```text
diff --git a/net/cloudflared/files/cloudflared.config b/net/cloudflared/files/cloudflared.config
index 324d3ecf3..11ca4a206 100644
--- a/net/cloudflared/files/cloudflared.config
+++ b/net/cloudflared/files/cloudflared.config
@@ -4,10 +4,14 @@ config cloudflared 'config'
        option token ''
        option config '/etc/cloudflared/config.yml'
        option origincert '/etc/cloudflared/cert.pem'
-       option edge-bind-address 'auto'
-       option protocol 'auto'
+       option edge_bind_address ''
+       option edge_ip_version ''
+       option grace_period ''
+       option protocol ''
        option region ''
-       option retries '5'
+       option retries ''
+       option tag ''
+       option metrics ''
        option loglevel 'info'
        option logfile '/var/log/cloudflared.log'

diff --git a/net/cloudflared/files/cloudflared.init b/net/cloudflared/files/cloudflared.init
index 2e511d8f5..a21b8adb0 100755
--- a/net/cloudflared/files/cloudflared.init
+++ b/net/cloudflared/files/cloudflared.init
@@ -26,10 +26,14 @@ start_service() {

        append_param_arg "config" "/etc/cloudflared/config.yml"
        append_param_arg "origincert" "/etc/cloudflared/cert.pem"
-       append_param_arg "edge-bind-address"
+       append_param_arg "edge_bind_address" "--edge-bind-address"
+       append_param_arg "edge_ip_version" "--edge-ip-version"
+       append_param_arg "grace_period"
        append_param_arg "protocol"
        append_param_arg "region"
        append_param_arg "retries"
+       append_param_arg "tag"
+       append_param_arg "metrics"
        append_param_arg "loglevel"
        append_param_arg "logfile"
```

Reply:
It's common to make mistakes, so no worries.

However, please don't touch stuffs that you don't use. You need `protocol` parameter, and you did test it, it's great. But for other parameters, seems no. You don't have to add them, leave them for others.
I'd suggest just removing edge-bind-address.

Reply:
Anyway I need to fix it ASAP, I dont want to ruin the day of some random guys...  
Thanks, thanks again and again for being such a nice and patient teacher. I really appreciate it.

Reply:
> > Yes, it can be auto, I did read the docs [here](https://developers.cloudflare.com/cloudflare-one/connections/connect-networks/configure-tunnels/tunnel-run-parameters/#edge-ip-version)
> 
> It's for edge-**ip**-version, not bind. The code seems to say no.

I didn't even realize it until you pointed it out... Oh my gosh

Reply:
Don't be frustrated, it's not the end of the world ;)